assets/css/style.css

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
indentation style is not consistent

Reply:
reset.css is not done by her. It is resetting helper.